docs/blueprints.rst

```diff
@@ -176,16 +176,24 @@ the `template_folder` parameter to the :class:`Blueprint` constructor::
 
     admin = Blueprint('admin', __name__, template_folder='templates')
 
-As for static files, the path can be absolute or relative to the blueprint
-resource folder.  The template folder is added to the searchpath of
-templates but with a lower priority than the actual application's template
-folder.  That way you can easily override templates that a blueprint
-provides in the actual application.
+For static files, the path can be absolute or relative to the blueprint
+resource folder.  
+
+The template folder is added to the search path of templates but with a lower 
+priority than the actual application's template folder. That way you can 
```

Comment:
In  [flask.templating.DispatchingJinjaLoader#_iter_loaders](https://github.com/pallets/flask/blob/master/flask/templating.py#L87) looks like the first loader checked is the main app loader and then it walks the blueprint loaders by the order they were registered

Also [this answer](http://stackoverflow.com/a/8737529/1631135) suggests the same.

> First it checks if the app's global loader, which is app.jinja_loader can locate the file. Failing that, it checks all application blueprints (in order of registration, AFAIK) for blueprint.jinja_loader in an attempt to locate the file. 

But i am still wondering about how i should phrase this